.github/workflows/ci-aqua-security-trivy-tests.yml

```diff
@@ -1,6 +1,26 @@
 name: ci-aqua-security-trivy-tests
 on:
   push:
+    paths-ignore:
+      - '.vscode/CODEOWNERS'
+      - '.vscode/*.json'
+      - 'compose/**'
+      - 'contrib/**'
```

Comment:
our canonical Dockerfile is located in `contrib`, this should be removed

Reply:
The description of Contrib says:
>The contrib directory contains scripts, images, and other helpful things which are not part of the core dgraph distribution. Please note that they could be out of date, since they do not receive the same attention as the rest of the repository.

The objective of the tests are related to the core code or helpful things we use time to time?

The thing is. If your PR change some code AND change something in contrib. The CI will be triggered.

Now, let's say I wanna update something in /contrib/systemd/ or  /contrib/jepsen. Are the CI covering those? So why run the CI at all?

If you need to update some dockerfile or docker compose. We will have an option to trigger manually if needed. But it is something we can think more about. Maybe have a granular control here instead ignoring the whole contrib file.